worlds/bk_sudoku/docs/setup_fr.md

```diff
@@ -0,0 +1,24 @@
+# Guide de configuration de BK Sudoku
+
+## Logiciels requis
+- [Bk Sudoku](https://github.com/Jarno458/sudoku)
+- Windows 8 ou supérieur
+
+## Concept général
+
+Il s'agit d'un client qui peut se connecter à n'importe quel slot du multi-monde, et qui vous permet de jouer au Sudoku pour débloquer des indices aléatoires pour les emplacements de ce slot.
+
+Comme le client Sudoku peut se connecter à n'importe quel slot, il n'est pas nécessaire de générer un YAML pour ce jeu car il ne génère pas de nouveaux slots dans la partie multi-monde.
+
+## Procédures d'installation
+
+Allez à la dernière version sur [BK Sudoku Releases](https://github.com/Jarno458/sudoku/releases). Téléchargez et extrayez le fichier `Bk_Sudoku.zip`.
+
+## Rejoindre une partie multi-monde
+
+1. Exécuter Bk_Sudoku.exe
+2. Saisissez le nom du slot auquel vous souhaitez vous connecter.
```

Comment:
As stated earlier, suggesting to change "slot" to "world" or maybe "le nom associé au monde"